core/invoker/src/main/scala/whisk/core/container/WhiskContainer.scala

```diff
@@ -67,6 +64,9 @@ class WhiskContainer(
     var lastLogSize = 0L
     private implicit val emitter: PrintStreamEmitter = this
 
+    /** HTTP connection to container. Initialized on /init. */
+    private var connection: HttpUtils = null
```

Comment:
😢 

Shall we make this `private var connection: Option[HttpUtils] = None` to deal with the different cases in a safer way?